deltachat-ffi/deltachat.h

```diff
@@ -4715,6 +4809,15 @@ int64_t          dc_lot_get_timestamp     (const dc_lot_t* lot);
  */
 #define DC_MSG_VIDEOCHAT_INVITATION 70
 
+
+/**
+ * The message is a webxdc instance.
+ *
+ * To send data to a webxdc instance, use dc_send_webxdc_status_update()
+ */
+#define DC_MSG_WEBXDC    80
```

Comment:
funny that the number is 80 like the HTTP port :D

Reply:
this is a sign, that cannot be coincidence! 🐇